kazoo/protocol/connection.py

```diff
@@ -605,9 +605,9 @@ def _connect_attempt(self, host, port, retry):
             return STOP_CONNECTING
         except (ConnectionDropped, KazooTimeoutError) as e:
             if isinstance(e, ConnectionDropped):
-                self.logger.warning('Connection dropped: %s', e)
+                self.logger.exception('Connection dropped: %s', e)
```

Comment:
How about adding the text `Zookeeper` or `Kazoo`? 

Eg. "Zookeeper Connection dropped: {e}"

Reply:
Sounds good I'll affect